kythe/extractors/bazel/extract.sh

```diff
@@ -65,7 +65,14 @@ fi
 # It is ok if targets fail to build. We build using --keep_going and don't
 # care if some targets fail, but bazel will return a failure code if any
 # targets fail.
-/kythe/bazelisk --bazelrc=/kythe/bazelrc "$@" || true
+if [ -n "$BAZEL_TARGET" ]; then
+  /kythe/bazelisk --bazelrc=/kythe/bazelrc "$@" "$BAZEL_TARGET" || true
+else
+  # If the user supplied a bazel query, execute it and run bazel, but we have to
+  # shard the results to different bazel runs because the bazel command line
+  # cannot take many arguments.
+  /kythe/bazelisk query "$BAZEL_QUERY" | xargs -t -L "${TARGETS_PER_INVOCATION}" /kythe/bazelisk --bazelrc=/kythe/bazelrc "$@" || true
```

Comment:
The other env vars we pass to this script are all prefixed with `KYTHE_` - we should probably do the same here for consistency.

Reply:
done